config/dev.webpack.config.js

```diff
@@ -32,6 +32,7 @@ const webpackProxy = {
     proxyVerbose: true,
     useCloud: (process.env?.USE_CLOUD === 'true'),
     ...useLocalChrome(),
+    routesPath: process.env.ROUTES_PATH || '/compliance/config/spandx.config.js',
```

Comment:
This could be:

```suggestion
    routesPath: process.env.ROUTES_PATH || resolve(__dirname, '../config/spandx.config.js'),
```

Then we can use it in both in and outside a container.